Source/Core/VideoCommon/FramebufferManager.h

```diff
@@ -124,8 +124,8 @@ class FramebufferManager final
   void EndOfFrame();
 
   // Writes a value to the framebuffer. This will never block, and writes will be batched.
-  void PokeEFBColor(u32 x, u32 y, u32 color);
-  void PokeEFBDepth(u32 x, u32 y, float depth);
+  void PokeEFBColor(u32 x, u32 y, u32 poke_data);
```

Comment:
While I have no plans for this, where we can, I'd like to keep a separation of emulation logic and the host gpu.

Can we leave this interface as is and just move your conversion calculation to `Renderer`?  Seems like it is just a passthrough right now.